Resources/doc/reference/ckeditor_medias.rst

```diff
@@ -42,6 +42,7 @@ Be sure to have the ``ivory/ivory_ckeditor.yml`` configuration file available. I
                 filebrowserImageUploadRouteParameters:
                     provider: sonata.media.provider.image
                     context: my-context # Optional, to upload in a custom context
+                    format: my-big # Optional, media format or original size returned to editor
```

Comment:
This format is needed for every media context/provider!

Reply:
I can add a new `ckeditor_format` parameter to configure it per form field. Is it okay for you?
```php
$formMapper->add('details', 'sonata_simple_formatter_type', array(
    'format' => 'richhtml',
    'ckeditor_context' => 'default', // optional
    'ckeditor_format' => 'big', // optional
));
```
Besides, what you can do at this moment is to have multiple ckeditor configurations.

Reply:
I just want to say, that it is risky to change the default format, because you have to define it for all media types, otherwise you will get an exception.

Please add a short note for this.

Reply:
@core23 I added subchapter about usage to documentation, please review.